app-admin/pwman3/pwman3-0.9.11.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_5 python3_6 )
+PYTHON_REQ_USE="sqlite?"
+inherit distutils-r1
+
+DESCRIPTION="A lightweight password-manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="mongodb mysql postgres +sqlite test"
+
+CDEPEND="
```

Comment:
One letter + DEPEND is reserved for future use.

Reply:
you prefer me to use COMMON isntead of CDEPEND?

btw, the Gentoo repo has many of the same CDEPEND...

Reply:
Yes, and yes, Gentoo is horribly bad quality product. You shouldn't use it ;-P.

Reply:
@mgorny I pushed a fixed, and I hope this does it now.